fabric_service_tokens.go

```diff
@@ -0,0 +1,48 @@
+package packngo
+
+import "path"
+
+type FabricServiceTokenType string
+
+const (
+	fabricServiceTokenBasePath                        = "/fabric-service-tokens"
+	FabricServiceTokenASide    FabricServiceTokenType = "a_side"
+	FabricServiceTokenZSide    FabricServiceTokenType = "z_side"
+)
+
+// FabricServiceTokenService interface defines available metro methods
+type FabricServiceTokenService interface {
+	Get(string, *GetOptions) (*FabricServiceToken, *Response, error)
+}
+
+// FabricServiceToken represents an Equinix Metal metro
+type FabricServiceToken struct {
+	ID               string                 `json:"id"`
+	Role             string                 `json:"role,omitempty"`
+	State            string                 `json:"state,omitempty"`
+	MaxAllowedSpeed  int                    `json:"max_allowed_speed,omitempty"`
+	ServiceTokenType FabricServiceTokenType `json:"service_token_type,omitempty"`
+	Connection       *Connection            `json:"interconnection,omitempty"`
+	ConnectionPort   *ConnectionPort        `json:"interconnection_port,omitempty"`
+	Organization     *Organization          `json:"organization,omitempty"`
+}
+
+func (f FabricServiceToken) String() string {
+	return Stringify(f)
+}
+
+// FabricServiceTokenServiceOp implements FabricServiceTokenService
+type FabricServiceTokenServiceOp struct {
+	client *Client
+}
+
+func (s *FabricServiceTokenServiceOp) Get(id string, opts *GetOptions) (*FabricServiceToken, *Response, error) {
```

Comment:
The `GET /fabric-service-tokens/{token_id}` operation has been dropped from the draft spec.

Reply:
Will remove the method.